application-skeleton-executor/src/main/java/eu/arrowhead/application/skeleton/executor/controller/ExecutorController.java

```diff
@@ -0,0 +1,58 @@
+package eu.arrowhead.application.skeleton.executor.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import eu.arrowhead.application.skeleton.executor.service.ExecutorService;
+import eu.arrowhead.common.CommonConstants;
+import eu.arrowhead.common.dto.shared.ChoreographerAbortStepRequestDTO;
+import eu.arrowhead.common.dto.shared.ChoreographerExecuteStepRequestDTO;
+import eu.arrowhead.common.dto.shared.ChoreographerExecutorServiceInfoRequestDTO;
+import eu.arrowhead.common.dto.shared.ChoreographerExecutorServiceInfoResponseDTO;
+
+@RestController
+public class ExecutorController {
```

Comment:
a TODO here about the baseUri, like
`// @RequestMapping("/executor")  // TODO: specify the base URI here`